content/applications/finance/accounting/payments.rst

```diff
@@ -30,114 +32,149 @@ At this point, the customer invoice or vendor bill is marked as :guilabel:`In pa
 the outstanding account is reconciled with a bank statement line, the invoice or vendor bill changes
 to the :guilabel:`Paid` status.
 
-The information icon near the payment line displays more information about the payment. You can
-access additional information, such as the related journal, by clicking :guilabel:`View`.
+The :icon:`fa-info-circle` information icon next to the payment line displays more
+information about the payment. To access additional information, such as the related journal, click
+:guilabel:`View`.
 
 .. image:: payments/information-icon.png
    :alt: See detailed information of a payment.
 
 .. note::
    - The customer invoice or vendor bill must be in the :guilabel:`Posted` status to register the
```

Comment:
Reminder that this isn't the case as of 18 and will need to be updated - more details here: https://www.odoo.com/odoo/project/967/tasks/3979071

Reply:
@dade-odoo Yes, sure, I added it to the list of updates for 18.